<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Articulos</class>
 <widget class="QDialog" name="Articulos">
  <property name="windowModality">
   <enum>Qt::WindowModality::WindowModal</enum>
  </property>
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1039</width>
    <height>578</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Productos</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QLabel" name="labelNombrePrecio">
     <property name="font">
      <font>
       <family>Utopia</family>
       <pointsize>16</pointsize>
       <italic>true</italic>
       <bold>true</bold>
      </font>
     </property>
     <property name="styleSheet">
      <string notr="true">
color: rgb(255, 25, 5);</string>
     </property>
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QTabWidget" name="tabWidget">
     <property name="tabPosition">
      <enum>QTabWidget::TabPosition::North</enum>
     </property>
     <property name="tabShape">
      <enum>QTabWidget::TabShape::Rounded</enum>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <property name="elideMode">
      <enum>Qt::TextElideMode::ElideMiddle</enum>
     </property>
     <property name="movable">
      <bool>false</bool>
     </property>
     <property name="tabBarAutoHide">
      <bool>false</bool>
     </property>
     <widget class="QWidget" name="tab">
      <attribute name="title">
       <string>General</string>
      </attribute>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QWidget" name="General" native="true">
         <layout class="QGridLayout" name="gridLayout">
          <property name="sizeConstraint">
           <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
          </property>
          <property name="leftMargin">
           <number>4</number>
          </property>
          <property name="topMargin">
           <number>4</number>
          </property>
          <property name="rightMargin">
           <number>4</number>
          </property>
          <property name="bottomMargin">
           <number>4</number>
          </property>
          <property name="spacing">
           <number>4</number>
          </property>
          <item row="5" column="3">
           <widget class="QDateEdit" name="dateEditUltimoPedido">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
          <item row="2" column="6">
           <widget class="QLineEdit" name="lineEditPvp"/>
          </item>
          <item row="7" column="4">
           <widget class="QLabel" name="label_12">
            <property name="text">
             <string>Foto</string>
            </property>
           </widget>
          </item>
          <item row="4" column="5" rowspan="3">
           <widget class="QLabel" name="labelFoto">
            <property name="sizePolicy">
             <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
              <horstretch>0</horstretch>
              <verstretch>0</verstretch>
             </sizepolicy>
            </property>
            <property name="text">
             <string>Sin foto</string>
            </property>
            <property name="scaledContents">
             <bool>false</bool>
            </property>
            <property name="openExternalLinks">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="label_11">
            <property name="text">
             <string>Ultimo pedido</string>
            </property>
           </widget>
          </item>
          <item row="7" column="7">
           <widget class="QPushButton" name="pushButtonBorrarFoto">
            <property name="text">
             <string>Borrar foto</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Codigo</string>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QLineEdit" name="lineEditCod">
            <property name="maxLength">
             <number>13</number>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QPushButton" name="pushButtonEtiqueta">
            <property name="text">
             <string>Crear etiqueta</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="label_2">
            <property name="text">
             <string>Descripcion</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="label_4">
            <property name="text">
             <string>IVA</string>
            </property>
           </widget>
          </item>
          <item row="2" column="1">
           <widget class="QLineEdit" name="lineEditIva"/>
          </item>
          <item row="2" column="3">
           <widget class="QLabel" name="label_14">
            <property name="text">
             <string>Costo</string>
            </property>
           </widget>
          </item>
          <item row="2" column="4">
           <widget class="QLineEdit" name="lineEditCosto">
            <property name="layoutDirection">
             <enum>Qt::LayoutDirection::LeftToRight</enum>
            </property>
            <property name="inputMask">
             <string/>
            </property>
            <property name="maxLength">
             <number>32767</number>
            </property>
            <property name="alignment">
             <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
            </property>
           </widget>
          </item>
          <item row="2" column="5">
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>P.V.P.</string>
            </property>
           </widget>
          </item>
          <item row="3" column="0">
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Stock</string>
            </property>
           </widget>
          </item>
          <item row="3" column="1">
           <widget class="QLineEdit" name="lineEditStock">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item row="3" column="2">
           <widget class="QPushButton" name="pushButtonVer">
            <property name="text">
             <string>Ver/Actualizar</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="3" column="3">
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>Minimo</string>
            </property>
           </widget>
          </item>
          <item row="3" column="4">
           <widget class="QLineEdit" name="lineEditMinimo"/>
          </item>
          <item row="3" column="5">
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>Maximo</string>
            </property>
           </widget>
          </item>
          <item row="3" column="6">
           <widget class="QLineEdit" name="lineEditMaximo"/>
          </item>
          <item row="4" column="0">
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>Pendientes pedido</string>
            </property>
           </widget>
          </item>
          <item row="4" column="1">
           <widget class="QLineEdit" name="lineEditPendientes">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="label_10">
            <property name="text">
             <string>Ultima venta</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDateEdit" name="dateEditUltimaVenta">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
            <property name="correctionMode">
             <enum>QAbstractSpinBox::CorrectionMode::CorrectToNearestValue</enum>
            </property>
            <property name="displayFormat">
             <string>yyyy-MM-dd</string>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Familia</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QLineEdit" name="lineEditCodFamila">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="labelFamilia">
            <property name="text">
             <string>Sin famila asignada</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="label_16">
            <property name="text">
             <string>Fabricante</string>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QLineEdit" name="lineEditCodFabricante">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="labelFabricante">
            <property name="text">
             <string>Fabricante desconocido</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0" colspan="10">
           <widget class="QPlainTextEdit" name="plainTextEdit"/>
          </item>
          <item row="1" column="1" colspan="6">
           <widget class="QLineEdit" name="lineEditDesc"/>
          </item>
          <item row="7" column="6">
           <widget class="QPushButton" name="pushButtonPonerFoto">
            <property name="text">
             <string>Añadir Foto</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="7" column="3">
           <widget class="QPushButton" name="pushButton_2">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="6" column="3">
           <widget class="QPushButton" name="pushButtonBuscarFamilia">
            <property name="maximumSize">
             <size>
              <width>40</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="autoDefault">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item row="8" column="0">
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Presentación</string>
            </property>
           </widget>
          </item>
          <item row="4" column="2">
           <widget class="QLabel" name="label_9">
            <property name="text">
             <string>Encargados</string>
            </property>
           </widget>
          </item>
          <item row="7" column="5">
           <widget class="QLineEdit" name="lineEditFoto">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>11</height>
             </size>
            </property>
            <property name="focusPolicy">
             <enum>Qt::FocusPolicy::NoFocus</enum>
            </property>
           </widget>
          </item>
          <item row="4" column="3">
           <widget class="QLineEdit" name="lineEditEncargados">
            <property name="enabled">
             <bool>false</bool>
            </property>
            <property name="text">
             <string>0</string>
            </property>
           </widget>
          </item>
          <item row="8" column="2">
           <widget class="QLabel" name="label_17">
            <property name="text">
             <string>Cantidad</string>
            </property>
           </widget>
          </item>
          <item row="8" column="1">
           <widget class="QComboBox" name="comboBoxFormato"/>
          </item>
          <item row="8" column="3">
           <widget class="QLineEdit" name="lineEditCantidad"/>
          </item>
          <item row="1" column="7" rowspan="5">
           <widget class="QTreeWidget" name="treeWidgetStockTiendas">
            <property name="contextMenuPolicy">
             <enum>Qt::ContextMenuPolicy::DefaultContextMenu</enum>
            </property>
            <column>
             <property name="text">
              <string notr="true">1</string>
             </property>
            </column>
           </widget>
          </item>
          <item row="0" column="7">
           <widget class="QCheckBox" name="checkBoxRemoto">
            <property name="text">
             <string>Cargar remoto</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_3">
      <attribute name="title">
       <string>Códigos auxiliares</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_6">
       <item row="0" column="0">
        <layout class="QGridLayout" name="gridLayout_5">
         <item row="0" column="0" colspan="2">
          <widget class="QTableView" name="tableViewAux"/>
         </item>
         <item row="2" column="0">
          <widget class="QPushButton" name="pushButtonAnadir">
           <property name="text">
            <string>Añadir</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QPushButton" name="pushButtonEliminar">
           <property name="text">
            <string>Eliminar</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0" colspan="2">
          <widget class="QLineEdit" name="lineEditAux">
           <property name="maxLength">
            <number>13</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tabVentas">
      <attribute name="title">
       <string>Ventas</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_3">
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Agrupar por:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QRadioButton" name="radioButtonVentasDia">
            <property name="text">
             <string>Dia</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonVentasMes">
            <property name="text">
             <string>Mes</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonVentasAno">
            <property name="text">
             <string>A&amp;ño</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QTableView" name="tableViewVentas"/>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="tab_2">
      <attribute name="title">
       <string>Compras</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout_4">
       <item row="1" column="0">
        <widget class="QTableView" name="tableViewCompras">
         <property name="selectionBehavior">
          <enum>QAbstractItemView::SelectionBehavior::SelectRows</enum>
         </property>
        </widget>
       </item>
       <item row="0" column="0">
        <widget class="QGroupBox" name="groupBox_2">
         <property name="title">
          <string>Ver por:</string>
         </property>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QRadioButton" name="radioButtonFacturas">
            <property name="text">
             <string>Facturas</string>
            </property>
            <property name="checked">
             <bool>false</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonMeses">
            <property name="text">
             <string>&amp;Meses</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioButtonAnos">
            <property name="text">
             <string>A&amp;ños</string>
            </property>
           </widget>
          </item>
         </layout>
        </widget>
       </item>
       <item row="3" column="0">
        <widget class="QPushButton" name="pushButtonVerFactura">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="text">
          <string>Ver factura</string>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="labelProveedor">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="pushButtonNuevo">
       <property name="text">
        <string>Nuevo</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonBorrar">
       <property name="enabled">
        <bool>false</bool>
       </property>
       <property name="text">
        <string>Borrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonModificar">
       <property name="text">
        <string>Modificar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonRefrescar">
       <property name="text">
        <string>Refrescar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonAnterior">
       <property name="text">
        <string>Anterior</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonSiguiente">
       <property name="text">
        <string>Siguiente</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCaducados">
       <property name="text">
        <string>Caducados</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="pushButtonCerrar">
       <property name="text">
        <string>Cerrar</string>
       </property>
       <property name="autoDefault">
        <bool>false</bool>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>lineEditCod</tabstop>
  <tabstop>lineEditDesc</tabstop>
  <tabstop>lineEditIva</tabstop>
  <tabstop>lineEditCosto</tabstop>
  <tabstop>lineEditPvp</tabstop>
  <tabstop>lineEditMinimo</tabstop>
  <tabstop>lineEditMaximo</tabstop>
  <tabstop>lineEditCodFamila</tabstop>
  <tabstop>pushButtonBuscarFamilia</tabstop>
  <tabstop>lineEditCodFabricante</tabstop>
  <tabstop>pushButton_2</tabstop>
  <tabstop>comboBoxFormato</tabstop>
  <tabstop>lineEditCantidad</tabstop>
  <tabstop>pushButtonEtiqueta</tabstop>
  <tabstop>pushButtonNuevo</tabstop>
  <tabstop>pushButtonBorrar</tabstop>
  <tabstop>pushButtonModificar</tabstop>
  <tabstop>pushButtonRefrescar</tabstop>
  <tabstop>pushButtonAnterior</tabstop>
  <tabstop>pushButtonSiguiente</tabstop>
  <tabstop>pushButtonCaducados</tabstop>
  <tabstop>pushButtonCerrar</tabstop>
  <tabstop>pushButtonVer</tabstop>
  <tabstop>pushButtonPonerFoto</tabstop>
  <tabstop>plainTextEdit</tabstop>
  <tabstop>treeWidgetStockTiendas</tabstop>
  <tabstop>lineEditEncargados</tabstop>
  <tabstop>lineEditPendientes</tabstop>
  <tabstop>tabWidget</tabstop>
  <tabstop>tableViewAux</tabstop>
  <tabstop>pushButtonAnadir</tabstop>
  <tabstop>pushButtonEliminar</tabstop>
  <tabstop>lineEditAux</tabstop>
  <tabstop>radioButtonVentasDia</tabstop>
  <tabstop>radioButtonVentasMes</tabstop>
  <tabstop>radioButtonVentasAno</tabstop>
  <tabstop>tableViewVentas</tabstop>
  <tabstop>tableViewCompras</tabstop>
  <tabstop>radioButtonFacturas</tabstop>
  <tabstop>radioButtonMeses</tabstop>
  <tabstop>radioButtonAnos</tabstop>
  <tabstop>pushButtonVerFactura</tabstop>
  <tabstop>pushButtonBorrarFoto</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>pushButtonCerrar</sender>
   <signal>clicked()</signal>
   <receiver>Articulos</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>1031</x>
     <y>570</y>
    </hint>
    <hint type="destinationlabel">
     <x>788</x>
     <y>488</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
